<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortcutCreatorDialog</class>
 <widget class="QDialog" name="ShortcutCreatorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New shortcut</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="actionLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="actionsRadioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Action:</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="actionsComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="scriptLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="scriptsRadioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Script:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="scriptsComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Shortcut:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KeySequenceEdit" name="sequenceEdit">
     <property name="text">
      <string>[Enter shortcut]</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="warningLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KeySequenceEdit</class>
   <extends>QPushButton</extends>
   <header>gui/customwidgets/keysequenceedit.h</header>
   <slots>
    <signal>edited()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ShortcutCreatorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShortcutCreatorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionsRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionsComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionsRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>scriptsComboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scriptsRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionsComboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scriptsRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>scriptsComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sequenceEdit</sender>
   <signal>edited()</signal>
   <receiver>ShortcutCreatorDialog</receiver>
   <slot>onShortcutEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onShortcutEdited()</slot>
 </slots>
</ui>
